core/org.openjdk.jmc.flightrecorder.rules.jdk/src/main/java/org/openjdk/jmc/flightrecorder/rules/jdk/general/FinalizersRunRule.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ *
```

Comment:
Added Datadog to all modified files.

Reply:
Not to all modified. The ones that were originally contributed by Oracle will keep having only the Oracle copyright, even if we modify them. Only do this for newly added files created by you (Datadog).

Reply:
Okay, this should hopefully be good now.

We seem to be quite inconsistent with regards to copyright headers, we often add Datadog for files that were just modified for example in 303031fa3d0238574e2f30a0d28b1c76b065f609. Maybe this could benefit from some clarification in the docs?